tests/Unit/Evolution/DiscontinuousGalerkin/SlopeLimiters/TestHelpers.hpp

```diff
@@ -0,0 +1,45 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <unordered_set>
+
+#include "Domain/Element.hpp"
+#include "Domain/ElementId.hpp"
+#include "Domain/Neighbors.hpp"
+#include "Domain/OrientationMap.hpp"
+
+namespace TestHelpers {
+namespace SlopeLimiters {
+
+// Construct a Neighbors object with one neighboring element.
+template <size_t VolumeDim>
+Neighbors<VolumeDim> make_neighbor_with_id(const size_t id) noexcept {
+  return {std::unordered_set<ElementId<VolumeDim>>{ElementId<VolumeDim>(id)},
+          OrientationMap<VolumeDim>{}};
+}
+
+// Construct an Element with one neighboring element in each direction.
+//
+// The optional argument specifies directions to external boundaries, i.e.,
+// directions where there is no neighboring element.
+template <size_t VolumeDim>
+Element<VolumeDim> make_element(
+    const std::unordered_set<Direction<VolumeDim>>&
+        directions_of_external_boundaries = {}) noexcept {
+  typename Element<VolumeDim>::Neighbors_t neighbors;
+  for (const auto dir : Direction<VolumeDim>::all_directions()) {
+    // Only add internal boundaries
+    if (not alg::found(directions_of_external_boundaries, dir)) {
```

Comment:
Use `directions_of_external_boundaries.count(dir) == 0`.  (C++20 adds a `contains` method to make this look less stupid.)  While performance isn't important here, using hash tables efficiently is usually not very hard so I think we should always do it.  (`alg::found` iterates over the container and checks each element.)